### PR TITLE
README and input variables improvements

### PR DESCRIPTION
**Pull request by mariux**
- doumentation is not complete yet... but better then nothing so far... will continue another time. no need to release yet..

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
 output "git_clone_url" {
-  value = github_repository.repository.git_clone_url
+  value       = github_repository.repository.git_clone_url
+  description = "URL that can be provided to git clone to clone the repository anonymously via the git protocol."
 }
 
 output "collaborators" {
-  value = github_repository_collaborator.collaborator
+  value       = github_repository_collaborator.collaborator
+  description = ""
```
Comment (soerenmartius) [inline outputs.tf]: add description here

File: outputs.tf [hunk 2]
```diff
@@ -1,23 +1,51 @@
...
 }
 
 output "projects" {
-  value = github_repository_project.repository_project
+  value       = github_repository_project.repository_project
+  description = ""
```
Comment (soerenmartius) [inline outputs.tf]: and here

File: outputs.tf (hunk 1 above)
Comment (mariux) [reply outputs.tf]: willl do in next step

File: outputs.tf (hunk 2 above)
Comment (mariux) [reply outputs.tf]: will do in next step

**Pull request merged**